src/core/worker.js

```diff
@@ -197,6 +197,126 @@ IPDFStreamRangeReader.prototype = {
 }
 
 /** @implements {IPDFStream} */
+function PDFNetworkStream(data, msgHandler) {
+  this.data = data;
+  let source = data.source;
+  this._contentLength = source.length;
+  this.msgHandler = msgHandler;
```

Comment:
Nit: Let's place `this.msgHandler = msgHandler;` right after the `this.data = data;` line, to group the properties that are set from the provided parameters.